### PR TITLE
[UI/UX] Cache HowLongtoBeat Data and move squares down

### PR DESCRIPTION
**Pull request by flavioislima**
Moved the squares with the info a bit down since it was causing a annoying layout shift when opening the game page, even with cached data.

Screenshot 2022-12-11 at 16 49 49

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by flavioislima** [comment]
> Tested here, works fine!

Can you approve it?

File: src/backend/howlongtobeat/utils.ts [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
 export async function getHowLongToBeat(
   title: string
 ): Promise<HowLongToBeatEntry | null> {
   try {
+    title = removeSpecialcharacters(title)
+
+    // check if we have a cached response
+    const cachedResponse = howLongToBeatStore.get(title)
+    if (cachedResponse) {
+      logInfo([`Using cached HowLongToBeat data for ${title}`, ''], {
```
Comment (arielj) [inline src/backend/howlongtobeat/utils.ts:15]: Just being nit picky here, but I think this is supposed to be used like:

```ts
logInfo(['Using cached HowLongToBeat data for' title], {
```

**Review by arielj — APPROVED** [review]
Looks good, I just added one comment.

Should this be merged against `stable` instead? seems like a small enhancement that could go in the next stable release

File: src/backend/howlongtobeat/utils.ts (hunk 1 above)
Comment (flavioislima) [reply src/backend/howlongtobeat/utils.ts:15]: I think I started one approach, changed it and left that empty string there. Good catch.

**Comment by flavioislima** [comment]
If I choose `stable` to merge it will bring changes from the main branch so I will do it in `main` and then cherry-pick to the `stable` branch

**Pull request merged**